### PR TITLE
Add modifications for ARM 64 ci

### PR DESCRIPTION
**Pull request by MichaelHudgins**
Begin to add required changes to bazelrc and envs to support ARM64 in the new script system.

### CODE REVIEW

**Comment by MichaelHudgins** [comment]
Needed to add a `release_linux_base` so arm could inherit from it.  Everything i believe is the same as x86 cpu except the crosstool which bazel did not like being defined twice.

**Comment by MichaelHudgins** [comment]
@angerson do you think we should put this one in official for now or over in devinfra?

**Comment by angerson** [comment]
I'd say to put it in Official with a WIP tag somewhere.

File: .bazelrc [hunk 1]
```diff
@@ -563,43 +567,46 @@ try-import %workspace%/.bazelrc.user
...
-test:release_cpu_linux --test_env=LD_LIBRARY_PATH
+test:release_linux_base --build_tests_only --keep_going --test_output=errors --verbose_failures=true
+test:release_linux_base --local_test_jobs=HOST_CPUS
+test:release_linux_base --test_env=LD_LIBRARY_PATH
 # Give only the list of failed tests at the end of the log
-test:release_cpu_linux --test_summary=short
+test:release_linux_base --test_summary=short
+
+# Use the Clang toolchain to compile
+build:release_cpu_linux --config=release_linux_base
```
Comment (angerson) [inline .bazelrc:608]: Should we change "cpu" to "x86" across all configs? Not necessarily in this PR.

File: .bazelrc [hunk 2]
```diff
@@ -682,6 +695,11 @@ test:linux_cuda_wheel_test_filters --test_tag_filters=gpu,requires-gpu,-no_gpu,-
 test:linux_cuda_wheel_test_filters --build_tag_filters=gpu,requires-gpu,-no_gpu,-no_oss,-oss_excluded,-oss_serial,-no_cuda11,-no_oss_py38,-no_oss_py39,-no_oss_py310
 test:linux_cuda_wheel_test_filters --test_lang_filters=py --test_size_filters=small,medium
 test:linux_cuda_wheel_test --config=linux_cuda_wheel_test_filters -- //tensorflow/... -//tensorflow/wheel/integration_testing/... -//tensorflow/compiler/tf2tensorrt/... -//tensorflow/compiler/xrt/... -//tensorflow/core/tpu/... -//tensorflow/lite/... -//tensorflow/tools/toolchains/...
+# ARM64 WHEEL
+test:linux_arm64_wheel_test_filters --test_tag_filters=-no_oss,-no_aarch64,-oss_excluded,-oss_serial,-gpu,-tpu,-benchmark-test,-v1only,-no_oss_py38,-no_oss_py39,-no_oss_py310
```
Comment (angerson) [inline .bazelrc:699]: Boy, this is convenient.

File: .bazelrc [hunk 3]
```diff
@@ -682,6 +695,11 @@ test:linux_cuda_wheel_test_filters --test_tag_filters=gpu,requires-gpu,-no_gpu,-
...
+test:linux_arm64_wheel_test_filters --build_tag_filters=-no_oss,-no_aarch64,-oss_excluded,-oss_serial,-gpu,-tpu,-benchmark-test,-v1only,-no_oss_py38,-no_oss_py39,-no_oss_py310
+test:linux_arm64_wheel_test_filters --test_lang_filters=py --test_size_filters=small,medium
+test:linux_arm64_wheel_test --config=linux_arm64_wheel_test_filters -- //tensorflow/... -//tensorflow/wheel/integration_testing/... -//tensorflow/compiler/tf2tensorrt/... -//tensorflow/compiler/xrt/... -//tensorflow/core/tpu/... -//tensorflow/lite/... -//tensorflow/tools/toolchains/...  -//tensorflow/go/... -//tensorflow/java/... -//tensorflow/core/grappler/optimizers:auto_mixed_precision_test_cpu -//tensorflow/core/grappler/optimizers:remapper_test_cpu -//tensorflow/core/kernels/image:resize_bicubic_op_test
```
Comment (angerson) [inline .bazelrc:702]: tensorflow/wheel/integration_testing is a typo I made. It should be `tensorflow/python`.

**Review by angerson — APPROVED** [review]
Couple of unimportant questions and one typo fix request.

File: .bazelrc (hunk 1 above)
Comment (MichaelHudgins) [reply .bazelrc:608]: Yeah that is a good idea, i'll add a bug to track

**Pull request merged**